snowflake/database/migrations/000002_create_pods_view.up.sql

```diff
@@ -30,6 +31,7 @@ CREATE VIEW IF NOT EXISTS pods (
   destinationPodNamespace || '/' || destinationPodName,
   throughput,
   reverseThroughput,
+  flowtype,
```

Comment:
but we still want to add the `sourceTransportPort` and `destinationTransportPort` as two separate fields to the view?

Reply:
ok I can do that

Reply:
done